confidant/routes/credentials.py

```diff
@@ -592,92 +592,95 @@ def create_credential():
                      correct format, or a required field was not provided.
     :statuscode 403: Client does not have access to create credentials.
     '''
-    if not acl_module_check(resource_type='credential', action='create'):
-        msg = "{} does not have access to create credentials".format(
-            authnz.get_logged_in_user()
+    with stats.timer('create_credential'):
+        if not acl_module_check(resource_type='credential', action='create'):
+            msg = "{} does not have access to create credentials".format(
```

Comment:
nit: it would be good to start using f-strings in confidant. no need to change it for now but at least code that we touch in the future or add.

Reply:
good point, will start focusing on this